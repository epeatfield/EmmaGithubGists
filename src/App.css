.App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}

.Search-Bar {
  padding: 50px;
}

.Submit-Button {
  height: 56px;
}

.Table-Header {
  background-color: #282c34;
  color:antiquewhite;
  font-size: 14px;
}


.Table-Cell {
  color: antiquewhite;
  font-size: 14px;
}

th.MuiTableCell-head {
  color: antiquewhite;
  font-weight: 500;
  line-height: 1.5rem;
}

div.Table {
  margin-top: 50px;
}

h1 {
  font-family: "Roboto", "Helvetica", "Arial", sans-serif;
}

div.MuiContainer-root{
  width: auto;
}

p.about-me {
  width: 50%;
  margin: auto;
  padding: 20px; 
  line-height: 1.6;
}
