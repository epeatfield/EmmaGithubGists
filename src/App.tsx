import React from 'react';
import { SearchPage } from './components/SearchPage';
import { Switch, Route, BrowserRouter as Router } from 'react-router-dom';
import { FavoritesList } from './components/FavoritesList';
import Header from './components/Header';
// import Error from './components/Error';
// import About from './components/About';

export default function App() {
  return (
    <div className="App">
      <Router>
        <Header/>
        <Switch>
          <Route exact component={SearchPage} path='/' />
          <Route exact component={FavoritesList} path='/favorites' />
          {/* <Route exact component={About} path='/about'/>
          <Route exact component={Error} path='/404'/> */}
        </Switch>
      </Router>
    </div>
  );

}
