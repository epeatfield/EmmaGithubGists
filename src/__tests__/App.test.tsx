import React from 'react';
import { render, screen } from '@testing-library/react';
import App from '../App';
import {shallow} from 'enzyme';

describe('<App />', () => {
  const mt = shallow(<App />);
  it('Test Snapshot', () => {
    expect(mt).toMatchSnapshot();
  });
});
