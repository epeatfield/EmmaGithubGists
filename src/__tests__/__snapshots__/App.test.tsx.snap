// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Test Snapshot 1`] = `ShallowWrapper {}`;
